src/main/java/io/paradoxical/dropwizard/swagger/bundles/SwaggerUIBundle.java

```diff
@@ -10,13 +11,17 @@
     private final SwaggerUIConfigurator swaggerUIConfigurator;
 
     public SwaggerUIBundle(@NonNull final SwaggerUIConfigurator swaggerUIConfigurator) {
-        super(SwaggerAssets.Assets);
+        this(SwaggerAssets.Assets, swaggerUIConfigurator);
+    }
+
+    public SwaggerUIBundle(@NonNull final AssetsDefinition assets, @NonNull final SwaggerUIConfigurator swaggerUIConfigurator) {
+        super(assets);
         this.swaggerUIConfigurator = swaggerUIConfigurator;
     }
 
     @Override
     public void run(final Environment environment) {
         super.run(environment);
-        swaggerUIConfigurator.configure(environment, environment.jersey().getResourceConfig());
+        swaggerUIConfigurator.configure(environment, environment.jersey());
```

Comment:
Yes, because the jersey environment for admin is not the same as the one for the main application. 
This keeps the swaggerUIConfigurator from having to know if its building admin or not